### ISSUE TITLE
[Bug]: JAWS on Chromium does not announce ComboBox option on arrow down

### ISSUE THREAD

**Issue by gfisher-vmsft** [microsoft/fluentui#26923]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Screen Reader: JAWS version 2023.2210.29 ILM
Chrome version 110.0.5481.104
Edge version 110.0.1587.46
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/v-grfisher/pen/poOgwrp

## Pre-Requisites
- Consecutive ComboBox options must have the same text length

## Steps to repro
- Focus the combobox, but do not expand the options list
- Use arrow key down to move to the next option

## Actual Behavior
- screen reader does not read the new option aloud if text length is the same as the previous option
- combobox is not expanded (out of scope?), and 

## Expected Behavior
- screen reader reads the option aloud
- combobox is expanded

## Notes
- If consecutive options have different text length, JAWS will properly read the option change with keyboard navigation. This is reproducible by modifying one of the option's text property in the codepen
- At some point, JAWS will switch to double reading options with different text length, but single read options with same text length
- JAWS on Firefox properly reads options
- NVDA and Narrator properly read the options on Chromium
- ComboBox on V9 does not have this issue

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by smhigley** [comment]
Chatted w/ @gfisher-vmsft separately, this is external to JAWS -- specifically this issue: https://github.com/FreedomScientific/VFO-standards-support/issues/379
